settings.json

```diff
@@ -3,5 +3,6 @@
   "aws_spot_instance_bid_usd": 0.015,
   "aws_spot_instance_type": "t2.micro",
   "aws_spot_instance_count": 2,
-  "aws_sqs_batch_size": 10
+  "aws_sqs_batch_size": 10,
+  "fetch_timeout": 10000
```

Comment:
I would not do this here, let the user define it in the consumer, because that is also an input field and not all consumers will necessarily have fetch code running.

Reply:
Sure. I will make a new branch + PR for this, because now the branch name doesn't make sense.